app/helpers/offers_helper.rb

```diff
@@ -0,0 +1,27 @@
+module OffersHelper
+  def offer_state_notice(offer_state)
+    case offer_state
+    when "proposed" then t(".proposed_an_offer")
```

Comment:
This is now failing the i18n-tasks, could you take a look please?

Reply:
Yeah, sorry for that. Fixed;